application/Module.php

```diff
@@ -760,6 +775,99 @@ public function noindexItemSet(ZendEvent $event)
         $this->noindexResourceShow($view, $view->itemSet);
     }
 
+    /**
+     * Serialize the API output to JSON-LD.
+     */
+    public function serializeApiOutputJsonLd(ZendEvent $event)
+    {
+        $renderer = $event->getTarget();
+        $model = $event->getParam('model');
+        $format = $event->getParam('format');
+        $payload = $event->getParam('payload');
+        $output = $event->getParam('output');
+
+        if ('jsonld' !== $format) {
+            return;
+        }
+
+        $eventManager = $this->getServiceLocator()->get('EventManager');
+
+        if ($payload instanceof RepresentationInterface) {
+            $args = $eventManager->prepareArgs(['jsonLd' => $output]);
+            $eventManager->trigger('rep.resource.json_output', $payload, $args);
```

Comment:
I'm not sure we need this event anymore since this PR allows modules to modify the output.

Reply:
Removing the event here would mean pushing this to a 5.0 so probably not justified.